ahk/templates/registery/reg_delete.ahk

```diff
@@ -1,4 +1,4 @@
 {% extends "base.ahk" %}
```

Comment:
These are showing as a file move from the templates/window directory.

Not a big deal, but if these files could be _added_. rather than moved, and the window files _deleted_, I think it makes the diff a bit more comprehensible.

Reply:
Actually, I created new file but VCS (git) supposed that it's moved. Should i commit these changes via 2 commiting instead 1?

First commit: File deletion
Second commit: File creation

> As I said before, if I do it one commit, git supposed that they are moved